test/helper.ts

```diff
@@ -43,50 +43,44 @@ export const getDocUri = (p: string) => {
 };
 
 export const updateSettings = (setting: any, value: any) => {
-	const yamlConfiguration = vscode.workspace.getConfiguration("yaml");
-    return yamlConfiguration.update(setting, value, false);
-}
+	const yamlConfiguration = vscode.workspace.getConfiguration('yaml');
+	return yamlConfiguration.update(setting, value, false);
+};
 
 export const resetSettings = (setting: any, value: any) => {
-	const yamlConfiguration = vscode.workspace.getConfiguration("yaml");
-    return yamlConfiguration.update(setting, value, false);
-}
+	const yamlConfiguration = vscode.workspace.getConfiguration('yaml');
+	return yamlConfiguration.update(setting, value, false);
+};
 
 export async function setTestContent(content: string): Promise<boolean> {
-	const all = new vscode.Range(
-		doc.positionAt(0),
-		doc.positionAt(doc.getText().length)
-	);
-	return editor.edit(eb => eb.replace(all, content));
+	const all = new vscode.Range(doc.positionAt(0), doc.positionAt(doc.getText().length));
+	return editor.edit((eb) => eb.replace(all, content));
 }
 
 export async function testCompletion(
 	docUri: vscode.Uri,
 	position: vscode.Position,
 	expectedCompletionList: vscode.CompletionList
 ) {
-
 	// Executing the command `vscode.executeCompletionItemProvider` to simulate triggering completion
 	const actualCompletionList = (await vscode.commands.executeCommand(
 		'vscode.executeCompletionItemProvider',
 		docUri,
 		position
 	)) as vscode.CompletionList;
 
-	const sortedActualCompletionList = actualCompletionList.items.sort((a, b) => (a.label > b.label) ? 1 : -1);
+	const sortedActualCompletionList = actualCompletionList.items.sort((a, b) => (a.label > b.label ? 1 : -1));
```

Comment:
This looks odd to me. Why would a return statement need parentheses around it?

Reply:
It seems that only ternary operator required to have parentheses in single line arrow functions, that required by prettier.

Reply:
I didn't realize that would work or that prettier likes it that way.